client/compatability/smart_wizard.css

```diff
@@ -0,0 +1,221 @@
+/*!
```

Comment:
@AshHimself does this have to be in /client? can it be dynamically imported instead?